workflow-templates/plugin-tests.yml

```diff
@@ -14,8 +14,8 @@ jobs:
   build:
     name: ${{ matrix.build_type }}
     runs-on: ubuntu-latest
-    container: discourse/discourse_test:slim${{ matrix.build_type == 'frontend' && '-browsers' || '' }}
-    timeout-minutes: 30
+    container: discourse/discourse_test:slim${{ startsWith(matrix.build_type, 'frontend') && '-browsers' || '' }}
+    timeout-minutes: 60
```

Comment:
Are there any plugin tests that need 60-minute timeout? 😃

Reply:
Right... I didn't mean to change that 🤦 Thanks!